docs/data-management-on-carbondata.md

```diff
@@ -284,6 +286,20 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
 	 ALTER TABLE employee SET TBLPROPERTIES (‘CACHE_LEVEL’=’Blocklet’)
 	 ```
 	 
+	 - **Support Flat folder**
+	 
+	  This feature allows all carbondata and carbonindex files to keep directy under tablepath. Currently all carbondata/carbonindex files written under tablepath/Fact/Part0/Segment_NUM folder and it is not same as hive/parquet folder structure. This feature makes all files written will be directly under tablepath, it does not maintain any segment folder structure.
```

Comment:
rephase to  `This feature allows all data files and index files to keep directy under tablepath`

Reply:
Can you also explain why this is needed, or what scenario user should consider to use this feature.

Reply:
ok